addons/web/static/tests/helpers/test_utils.js

```diff
@@ -27,6 +27,30 @@ odoo.define('web.test_utils', async function (require) {
     const testUtilsPivot = require('web.test_utils_pivot');
     const tools = require('web.tools');
 
+    QUnit.begin(() => {
+        // alt attribute causes issues with scroll tests. Indeed, alt is
+        // displayed between the time we scroll to the bottom of a thread
+        // and the time we assert for the scroll position. The src
+        // attribute is removed as well to make sure images won't
+        // trigger a GET request on the server.
+        function replaceAttr(attrName, prefix, element) {
+            const attrKey = `${prefix}${attrName}`;
+            const attrValue = element.getAttribute(attrKey);
+            element.removeAttribute(attrKey);
+            element.setAttribute(`${prefix}data-${attrName}`, attrValue);
+        }
+        const attrsToRemove = ['alt', 'src'];
+        const attrPrefixes = ['', 't-att-', 't-attf-'];
+        const templates = new DOMParser().parseFromString(session.owlTemplates, "text/xml");
+        for (const attrName of attrsToRemove) {
+            for (const prefix of attrPrefixes) {
+                for (const element of templates.querySelectorAll(`*[${prefix}${attrName}]`)) {
+                    replaceAttr(attrName, prefix, element);
+                }
+            }
+        }
+        session.owlTemplates = templates.documentElement.outerHTML;
+    });
```

Comment:
is this has to be done in web, this can be directly done in setup.js:
 ```js
const templates = await loadFile(templatesUrl);
// here
window.__OWL_TEMPLATES__ = processTemplates(templates);
```

Reply:
I believe `setup.js` has only been introduced in V15. I'll definitly do that when fw-porting !